samples/TempLCD/GroveLCDLib.js

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2016, Intel Corporation.
+// JavaScript library for the Grove LCD
+
+export function GroveLCD() {
```

Comment:
@grgustaf maybe what I pointed out is off-the-topic, I totally get what the code is trying to show about how to import 2 files, what I am saying that we could've chosen a much simpler module, and maybe that's what we should do, where if someone is looking at our example, they don't have to understand all these I2C code that doesn't totally make sense to read, and it just defeats the purpose.